spec/javascript/support/puppeteer/steps/sharedSteps.js

```diff
@@ -8,10 +8,10 @@ const loginAsAgent = async (page) => {
   await page.click('.sign-in-btn')
   await page.waitForNavigation()
 
-  // Sale force login
+  // Salesforce login
   await page.waitForSelector('#username_container')
-  await page.type('#username', process.env.E2E_SALESFORCE_USERNAME)
-  await page.type('#password', process.env.E2E_SALESFORCE_PASSWORD)
+  await page.type('#username', 'dahlia-leasing-agent@exygy.com.full')
+  await page.type('#password', 'oBC12FPxGf3I')
```

Comment:
whoops. I've reset the password, but let's never ever put a real password into a file that's in source control even if we don't intend to commit it. that's why we have the env vars.

Reply:
weird.. how the password ended up here? did we force a push ? I cannot find the commit

Reply:
I removed it soon after it went up - the reason it was there in the first place is that I was having some issues getting the env vars to work locally, so to speed up development i used them directly, intending not to commit them at all! i managed not to commit the other env vars but these slipped by me 🤦‍♀️ 🙄 ...i need a combo face palm/eye roll emoji....